docs/LEARNING.md

```diff
@@ -1,2 +1,9 @@
 # Learning
 
+ARM64 assembly is a simple assembly language. You might be able to start off with [a reference card](https://courses.cs.washington.edu/courses/cse469/19wi/arm64.pdf).
+
+ARM also provides [digital reference](https://developer.arm.com/documentation) on the ARM64 (AArch64/ARMv8) architecture.
+
+I hope you enjoy learning ARM assembly and working through the exercises. This is a gateway to getting to know your phone or ARM64 based development hardware (the more recent Raspberry Pi models are an example).
+
+If you get stuck with any aspect of your learning or running the tests, the track maintainer Roland (metivier.roland@chlorophyt.us), is happy to help you directly.
```

Comment:
He is? 😁😁😁?

Reply:
I don't think there'll be massive amounts of students soon, but you might want to reconsider this :)

Reply:
An alternative would be to use Gitter: https://gitter.im/

Reply:
> An alternative would be to use Gitter: https://gitter.im/

How would I set this up? I've used Gitter before. I'm on `exercism/support` with my Matrix/Gitter account for now.

Reply:
If you're interested in doing this, I think @iHiD could create a `exercism/arm64-assembly` channel, correct?

Reply:
> If you're interested in doing this, I think @iHiD could create a `exercism/arm64-assembly` channel, correct?

I don't think I am interested in this enough just yet after giving it a little bit of thought. I'll see how things go.

Reply:
Okay sure!